ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -52,11 +51,11 @@
 ##########################
 # Additional build tools #
 ##########################
-- name: Install additional build tools if NOT RHEL 8
+- name: Install additional build tools if NOT RHEL8 and above 
   package: "name={{ item }} state=latest"
-  with_items: "{{ Additional_Build_Tools_NOT_RHEL8 }}"
+  with_items: "{{ Additional_Build_Tools_NOT_RHEL8Plus }}"
```

Comment:
We probably need to go through and sanitise some of these names. This one should probably be `Additional_Build_Tools_RHEL6_RHEL7` now but that doesn't need to be changed in this PR